.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: Cypress Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies
```

Comment:
might also need 

```
      - uses: actions/setup-node@v1
        with:
          node-version: 16
```

Reply:
i would prefer to have an explicit node setup. I think on other platforms than ubuntu 20.04 this won't work